src/Appwrite/Event/Build.php

```diff
@@ -97,7 +153,11 @@ public function trigger(): string|bool
             'project' => $this->project,
             'resource' => $this->resource,
             'deployment' => $this->deployment,
-            'type' => $this->type
+            'type' => $this->type,
+            'template' => $this->template,
+            'providerCommitHash' => $this->providerCommitHash,
+            'providerTargetUrl' => $this->providerTargetUrl,
+            'providerContribution' => $this->providerContribution
```

Comment:
aren't these properties supposed to be part of the deployment document itself?

Reply:
Good concern, will have to ask Khushboo. Also commitHash might be related to our redeploy issue we wanted to address - not fixed yet.

Reply:
Removed from event, it's already on deployment. Using it from there instead